tests/sql/operators/test_merge.py

```diff
@@ -109,7 +109,10 @@ def run_merge(target_table: Table, source_table: Table, merge_parameters, mode):
 )
 @pytest.mark.parametrize(
     "database_table_fixture",
-    [{"database": Database.SNOWFLAKE}, {"database": Database.BIGQUERY}],
+    [
+        {"database": Database.SNOWFLAKE},
+        {"database": Database.BIGQUERY},
+    ],
```

Comment:
@pankajkoti We should have tests for Redshift as well right?

Reply:
It was decided that merge operator should be in a separate PR and hence the tests would go in that PR.